package.json

```diff
@@ -9,7 +9,7 @@
     "client": "webpack -d --watch",
     "build_client": "webpack",
     "generateData": "node helper/dataGeneration-Mongo.js",
-    "seedMongo": "mongoimport --db=images --collection=urls --type=csv --headerline --file=mongoImages.csv"
+    "seedMongo": "mongoimport --db=images --collection=urls --type=tsv --headerline --file=mongoImages.tsv"
   },
```

Comment:
i'd love to learn more about tsv!  what are you reasons for using it?

Reply:
Here is a sample url for a picture: https://res.cloudinary.com/dq3iywusm/image/upload/w_400,h_400/SDC/400_he1hru.jpg

If you notice the part that says 'w_400,h_400' mongoimport broke the url at the comma so I couldn't upload it as a csv file, but had to make it a tsv file.

Reply:
Here is a sample image url:

https://res.cloudinary.com/dq3iywusm/image/upload/w_400,h_400/SDC/400_he1hru.jpg

With mongoimport, it broke up the url at the comma in the 'w_400,h_400' section so I had to change the file type from a csv to a tsv.